Assets/ScriptableObjects/AttackData.cs

```diff
@@ -23,4 +23,6 @@ public class AttackData : ScriptableObject
     public float projectileRange;
     public bool destroyProjectileOnHit;
 
+    public bool possessTarget;
+
 }
```

Comment:
Personal preference - could we reserve Assets/ScriptableObjects for assets instead of scripts?